src/jdk.jlink/share/classes/jdk/tools/jlink/internal/Platform.java

```diff
@@ -50,8 +50,9 @@ public static Platform parsePlatform(String platformString) {
         OperatingSystem os = OperatingSystem.valueOf(osName.toUpperCase(Locale.ROOT));
 
         archName = platformString.substring(index + 1);
-        // Alias architecture "amd64" to "X64"
-        archName = archName.replace("amd64", "X64");
+        // Alias architecture "amd64" to "X64" and "ppc64le" to "ppc64"
+        archName = archName.replace("amd64", "X64")
+                           .replace("ppc64le", "ppc64");
```

Comment:
I'd prefer it to be:

```suggestion
        // Alias architecture names, if needed
        archName = archName.replace("amd64", "X64");
        archName = archName.replace("ppc64le", "PPC64");
```

So that other arches would add here more naturally.

(Also convert to upper-case right away, to make the subsequent `archName.toUpperCase` no-op).

Reply:
Makes sense. Thanks for the review!